src/styles/colors.ts

```diff
@@ -5,6 +5,7 @@ export const colors = {
   formal: '#efecd6',
   primary: '#1a0259',
```

Comment:
And this one is `primary-background`, can you rename it, please?

Reply:
Changed